server.cjs

```diff
@@ -1,5 +1,7 @@
+/* c8 ignore start */
 const express = require('express');
 
 const app = express();
 app.use(express.static('./dist-vue'));
 app.listen(4185);
+/* c8 ignore stop */
```

Comment:
Example server stub, ignoring for test coverage as it's the most basic express usage possible, so no need to deal with mocking this for tests, and it is simple specifically so people can easily replace it. Adding c8 ignores so they can more easily add coverage to the server.js if it has more custom logic.